### ISSUE TITLE
UI glitches in v0.3.0

### ISSUE THREAD

**Issue by slothbag** [retsamknaps/Aktie#18]
Good news is the blank listboxes is fixed, I can actually see text now.  The bad news is the content is completely screwed.. i.e. 
- the Communities listbox shows "Test" for every entry.
- The posts section shows the same line repeated all the way down the screen
- Sometimes when you mouse over the items the content changes to something else and repeats all the way down the screen.

**Comment by retsamknaps** [comment]
Oh no!  We used Ubuntu 64 bit 14.04 with java 1.8.0_65.  Can you compare?

Did you try to upgrade an old install, or create a new one?  Can you get the sha256 checksum of the 
aktie_run_dir/lib/swt*.jar  file?

**Comment by slothbag** [comment]
Ubuntu 64 bit 15.10 java 1.8.0_45.

I just tried deleting the aktie_run_dir and let it run from fresh, same problem.

```
6bb54b6a4c0648b1aeafbfa0b8823c996f793a88de9147b934b6969830ee0ca5  swt_linux.jar
9943dc5b611ed9362c79dd80d803b3608dc66ff73df9ab558e6db802416c7542  swt_osx.jar
5f545b2865dd7c173bf2f5a20e87faab67b826894548f2a70e71824056b21761  swt_win_64.jar
ccedadbb4473bb44c0fab1e4dba15abb570454bdebca50c6c6c997d71cc34b97  swt_win.jar
```

I also tried in Ubuntu Mate desktop and Unity.. same problem.

**Comment by retsamknaps** [comment]
Thank you.  Can you try to restart aktie with this set?

export SWT_GTK3=0

**Comment by slothbag** [comment]
Nice find! that seems to have done the trick.

**Comment by retsamknaps** [comment]
Should be fixed in 0.4.5
